tests/sentry/auth/test_idpmigration.py

```diff
@@ -38,7 +38,7 @@ def test_verify_account(self):
             == f"auth:one-time-key:{verification_key}"
         )
         assert response.status_code == 302
-        assert response.url == "/auth/login/"
+        assert response.templates[0].name == "sentry/idp_email_verified.html"
```

Comment:
we'll want to make an additional request after this one (or combine it with the other test), to test the full flow, I think (from here to user login).

Reply:
yes. ah I see we're in test_idpmigration here, did we intend for this file to include integration tests as well?

Reply:
(i would be in favor of moving this test to `test_auth_organization_login`)

Reply:
i posted my comment earlier but i left it in pending without realizing 😅. I think `test_idpmigration` should just have unit tests since the integration tests are in `test_auth_organization_login.py`. like stated above, ill add this flow to the test i have in `test_auth_organization_login.py` once this is merged but maybe this test could stay here since the purpose of this test is to see `idp_confirm_email()` is setting the verification key in the django session and redirecting to the right page

Reply:
got it. to make it more clear it's a unit test, you could instead of using a request, call the function directly and mock out what you need, although I think it would be redundant with the integration tests.

Reply:
yeah that makes sense, should i remove it here or in the other pr once its added to the integration test?

Reply:
other pr is fine